Assets/Scripts/Models/Inventory/InventoryManager.cs

```diff
@@ -27,6 +27,11 @@ public InventoryManager()
 
     public bool PlaceInventory(Tile tile, Inventory inventory)
     {
+        tile = World.Current.GetFirstTileWithValidInventoryPlacement(3, tile.X, tile.Y, tile.Z);
```

Comment:
I'm not sure this should be the default behavior. A machine outputing to a single spot should jam and stop if the direct inventory output is full